tests/unit_tests/test_wave_06.py

```diff
@@ -109,8 +114,24 @@ def test_swap_best_by_category():
         my_priority="Clothing",
         their_priority="Decor"
     )
-
-    raise Exception("Complete this test according to comments below.")
+    # Assert
+    assert result, "Expected swap_best_by_category to return truthy value"
```

Comment:
You could do `assert result is True` to make sure that an actual `True` comes out of the function instead of just a truthy value